Makefile

```diff
@@ -53,8 +53,10 @@ E2E_S2I_IMAGE_TAG ?= task-s2i:latest
 # (fully qualified) container image passed to s2i task IMAGE param
 E2E_S2I_PARAMS_IMAGE ?= $(IMAGE_BASE)/${E2E_S2I_IMAGE_TAG}
 
+E2E_S2I_LANGUAGE ?= java
+
 # s2i end-to-end test pipeline params, the git repository URL and revision
-E2E_S2I_PARAMS_URL ?= https://github.com/cpuguy83/go-md2man.git
+E2E_S2I_PARAMS_URL ?= https://github.com/sclorg/s2i-php-container.git #https://github.com/sclorg/s2i-perl-container.git #https://github.com/sclorg/s2i-ruby-container.git #https://github.com/cpuguy83/go-md2man.git  #https://github.com/sclorg/django-ex          #https://github.com/HouariZegai/Calculator.git 
```

Comment:
Let's leave this line as it was before, and open an issue with the project repos (and official builder images) to amend the project later on.